### DISCUSSION TITLE
Viscidus freezing

### DISCUSSION THREAD

**Discussion by Plasmashield** [chromiecraft/chromiecraft#4240] [Suggestions]
I suggest reducing the number of attacks needed to freeze Viscidus. Since the raid has been changed from 40 to 25, I suggest reducing the number of frost attacks needed to freeze from 200 to about 100. Same as for poison bolts on Huhuran.

**Comment by ishidinho** [comment]
Really good points, in my eyes.

I would suggest a linear scaling for both abilities compared to the raid size:
* Viscidus stacks: 125 (down from 200)
* Huhuran Poison Bolts: 9 or 10 (down from 15)

**Comment by balleny** [comment]
actually, surprised it was applied for huhu only, but not for visc (huhu is already reduced to 9)
